Makefile

```diff
@@ -81,7 +81,7 @@ bootstrap:
 
 .PHONY: lint
 lint:
-	find . -name "*.ts" | grep -v /dist/ | grep -v /node_modules/ | grep -v "\.d\.ts" | xargs ${ROOTDIR}/node_modules/.bin/eslint
+	${ROOTDIR}/node_modules/.bin/eslint -c .eslintrc.js -f tap --ext .ts .
```

Comment:
While including `-f tap` may be useful for testing, it changes the output of this make rule and should probably be removed. Here and below.

Reply:
This was included more for illustration, but I'll remove the explicit output for the final version.